File: app/src/main/java/com/example/ela2/Submenu2_3.java
package com.example.ela2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;

public class Submenu2_3 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_submenu23);
    }

    public void ImageClick1 (View view){
        Intent browserIntent1=new Intent (Intent.ACTION_VIEW, Uri.parse("https://drive.google.com/file/d/1vUVK_BNdXCOZO9N3x_sRN0UisU1eRSjw/view?usp=sharing"));
        startActivity(browserIntent1);
    }

    public void ButtonClick1 (View view){
        Intent browserIntent2=new Intent (Intent.ACTION_VIEW, Uri.parse("https://drive.google.com/file/d/1LT_qRvciq6kgEKGmjnOr3SBQUVYMQLDP/view?usp=sharing"));
        startActivity(browserIntent2);
    }

}

File: app/src/main/java/com/example/ela2/Submenu2_3.kt
package com.example.ela2

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.ela2.databinding.ActivitySubmenu23Binding

class Submenu2_3 : AppCompatActivity() {
    private lateinit var binding: ActivitySubmenu23Binding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivitySubmenu23Binding.inflate(layoutInflater)
        setContentView(binding.root)
    }

    fun ImageClick1(view: View) {
        AssetHelper.openPdfFromAssets(this, "ArduinoNANO picture.pdf")
    }

    fun ButtonClick1(view: View) {
        AssetHelper.openPdfFromAssets(this, "ArduinoNanoManual.pdf")
    }
}
